src/Outsystems.SetupTools/Functions/Get-OSPerfTuningStatus.ps1

```diff
@@ -0,0 +1,309 @@
+            WriteNonTerminalError -Message "The current user is not Administrator or not running this script in an elevated session"
+
+            $installResult.Success = $False
+            $installResult.ExitCode = -1
+            $installResult.Message = 'The current user is not Administrator or not running this script in an elevated session'
+
+            return $installResult
+        }
+ 
+        $PerfTuning.Sections.ProcessSchedulingConfig = CreatePerfTuningStatus -Title "Setting Windows processor scheduling priority to 'background services'" `
+            -ScriptBlock `
+            {
+                $ShouldBeSkipped = $True
+                $Messages = @()
+
+                try
+                {
+                    $Value = RegRead -Path "HKLM:\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\PriorityControl" -Name "Win32PrioritySeparation" -Type "Dword"
+
+                    $ShouldBeSkipped = ($Value -eq 24)
+
+                    if ($ShouldBeSkipped)
+                    {
+                        $Messages = @("Already set to 'background services'.")
+                    }
+                    else
+                    {
+                        $Messages = @("Not set to 'background services'.")
+                    }
+                }
+                catch
+                {
+                    $Messages = @("Unable to determine current value. Skipping.")
+                }
+
+                # No current scenarios where we should skip this
+                return $(CreateResult -ShouldBeSkipped $ShouldBeSkipped -Messages $Messages)
+            }
+
+        $PerfTuning.Sections.NETConfig = CreatePerfTuningStatus -Title ".NET upload size limits and execution timeout configuration" `
+            -ScriptBlock `
+            {
+                $ShouldBeSkipped = $True
+                $Messages = @()
+
+                try
+                {
+                    $NETConfig = $(GetDotNetLimits)
+
+                    $CurrentMaxRequestLength = $NETConfig.SystemWeb.HttpRuntime.maxRequestLength
+                    $CurrentMaxRequestLengthInMB = "$($CurrentMaxRequestLength/1024) MB"
+                    $CurrentExecutionTimeout = $NETConfig.SystemWeb.HttpRuntime.executionTimeout
+
+                    # upload size limits default is 4096 KB
+                    #   https://docs.microsoft.com/en-us/dotnet/api/system.web.configuration.httpruntimesection.maxrequestlength
+                    $DefaultMaxRequestLength = 4096
+                    $DefaultMaxRequestLengthInMB = "$($DefaultMaxRequestLength/1024) MB"
+
+                    # execution timeout default is 110 seconds
+                    #   https://docs.microsoft.com/en-us/dotnet/api/system.web.configuration.httpruntimesection.executiontimeout
+                    [TimeSpan]$DefaultExecutionTimeout = "00:01:50"
```

Comment:
(same as above) We should move this to a constant in src/Outsystems.SetupTools/Lib/Constants.ps1